src/libraries/System.Security.Cryptography/System.Security.Cryptography.sln

```diff
@@ -0,0 +1,72 @@
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Did you make sure that the sln builds correctly inside VS if you try to build the ref/src/test/ csproj?

Reply:
Opening the .sln and firing off Ctrl+Shift+B:

`========== Build: 7 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========`

Reply:
Great!